concourse/parameters/ci/ci.yml

```diff
@@ -2,4 +2,4 @@ aws_region: eu-west-1
 ecr_registry_id: "172025368201"
 concourse_ci_role_arn: arn:aws:iam::430354129336:role/govuk-ci-concourse
 concourse_ecr_role_arn: arn:aws:iam::172025368201:role/pull_images_from_ecr_role
-govuk_infrastructure_branch: main
+govuk_infrastructure_branch: use-govuk-ruby-base-image
```

Comment:
Why do we need to use a specific branch? Isn't the goal to use main once everything is merged?